sketchlib/links.js

```diff
@@ -36,6 +36,12 @@ export const LINKS = [
     thumbnail: "thumbnails/coral-tile-editor.png",
     year: "2025",
   },
+  {
+    link: "./lab/DoublePendulum",
+    title: "Double Pendulum",
+    is_lab: true,
+    year: "2016, 2025",
```

Comment:
At least for the 2025 sketches, look over old PRs and find key months so I know how to order them in the list.